Makefile

```diff
@@ -1471,7 +1473,7 @@ ifeq ($(PLATFORM), OS_SOLARIS)
 	ROCKSDBJNILIB = librocksdbjni-solaris$(ARCH).so
 	ROCKSDB_JAR = rocksdbjni-$(ROCKSDB_MAJOR).$(ROCKSDB_MINOR).$(ROCKSDB_PATCH)-solaris$(ARCH).jar
 	JAVA_INCLUDE = -I$(JAVA_HOME)/include/ -I$(JAVA_HOME)/include/solaris
-	SHA256_CMD = digest -a sha256
+	SHA1_CMD = digest -a sha1
```

Comment:
I did run into this issue a few weeks back, due to ssl version shipped with OS X being too old. You could continue using sha256 on mac if you switch the command to `openssl sha256 -r` on OSX (or switch to a newer version openssl from homebrew, which we don't need to if we don't want to introduce another dependency). And yes, unfortunately, the cut command also needs to change for OSX :( , as you did. I didn't know the Makefile's code well enough at that time to muster the courage to send out a PR.